### PR TITLE
docs: Add Easypanel as a deployment method

### PR DESCRIPTION
**Pull request by berksmbl**
[referral](https://github.com/ClickHouse/ClickHouse/pull/50851)

### CODE REVIEW

**Comment by robot-ch-test-poll1** [comment]
This is an automated comment for commit 014dc649d4a3c12277ce46018814ca0214d0dcd4 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/1237/014dc649d4a3c12277ce46018814ca0214d0dcd4/ci_running.html)
The overall status of the commit is 🟢 success

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>🟢 success</td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>🟢 success</td></tr>
</table>

File: sidebars.js [hunk 1]
```diff
@@ -95,6 +95,7 @@ const sidebars = {
         'en/integrations/data-ingestion/kafka/index',
         'en/integrations/data-ingestion/etl-tools/dbt/index',
         'en/integrations/data-ingestion/redshift/index',
+        'en/integrations/deployment/easypanel/index',
```
Comment (mshustov) [inline sidebars.js:98]: We keep this list minimalistic not to overload the menu.
```suggestion
```

Comment (berksmbl) [reply sidebars.js]: I hid it under the sub menu.

Comment (berksmbl) [reply sidebars.js]: @mshustov I hid it under the sub menu.

**Pull request merged**